scripts/dragonboard410c/setup.sh

```diff
@@ -21,6 +21,8 @@ print_help() {
         echo "        Build and install the minimum required version of opencv for the dnn example (3.4.1)"
         echo "-op|--opencvpath"
         echo "        Path to opencv installation "
+		echo "-ur|--udevrules"
```

Comment:
For all the scripts, formatting looks a bit odd. To many spaces/tabs in some cases

Reply:
solved. thanks